js/interactive-guides/microprofile-config/microprofile-config-callback.js

```diff
@@ -10,7 +10,91 @@
 *******************************************************************************/
 var microprofileConfigCallBack = (function() {
 
+<<<<<<< HEAD
```

Comment:
Hmm, why do we have the merge lines here? Have Steven moved the codes to guides-common?

Reply:
Dang!   I'm not sure why those lines are there either.  I had no indication that there was a merge conflict when I the pushed.   I will remove the lines as was done in Steven's work item.

Reply:
Dang!   I'm not sure why I was not notified of the merge conflict!.....I'm removing the lines as Steven had done.